code/modules/mob/living/carbon/alien/alien.dm

```diff
@@ -33,6 +40,8 @@
 	create_internal_organs()
 
 	. = ..()
+	if(speed != 0)
+		add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/alien_varspeed, multiplicative_slowdown = speed)
```

Comment:
This seems to be a duplicate of `simplemob_varspeed`
Could you set `speed` to be a `/living` var, and generalize this?